mathics/builtin/drawing/image.py

```diff
@@ -146,6 +148,8 @@ def extract(im, evaluation):
 
 class ImageImport(_ImageBuiltin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/ImageImport.html</url>
```

Comment:
There is a bug in the doc handling that lowercases interior I's so this link turns out to be invalid. 

I guess we will handle that in another PR.